docs/monitoring-usage.rst

```diff
@@ -278,3 +278,110 @@ follows::
+>>> resource = client.resource('gce_instance', instance_id='my-instance', zone='us-central1-f')
+
+Please refer to the `Metrics`_ documentation for more information.
+
+With a Metric and Resource specified, the :class:`~gcloud.monitoring.timeseries.MetricStream`
+can be used to write  :class:`~gcloud.monitoring.timeseries.Point` values.
+:class:`~gcloud.monitoring.timeseries.MetricStream` is a helper class that helps create
+:class:`~gcloud.monitoring.timeseries.TimeSeries` objects with the same Metric and Resource types.
+
+When writing points with :class:`~gcloud.monitoring.timeseries.MetricStream`, you specify
+points containing a value of the type that matches the *value_type* specified in the associated
+:class:`~gcloud.monitoring.metric.MetricDescriptor`. The point must also contain a time interval.
+The point's time interval must be later than any points already created for the same
+:class:`~gcloud.monitoring.timeseries.TimeSeries`
+(where :class:`~gcloud.monitoring.timeseries.TimeSeries` with matching :class:`~gcloud.monitoring.metric.Metric` and :class:`~gcloud.monitoring.resource.Resource` types are considered the same).
+ The end
+time of the interval must not be more than 24 hours in the past or more than five minutes in the
+future::
+
+    >>> start = datetime.utcnow()
+    >>> end = datetime.utcnow() - datetime.timedelta(minutes=5)
+    >>> value = get_num_404s_in_last_five_minutes() # get value from a hypothetical call
+    >>> metric_stream = client.metric_stream(metric, resource)
+    >>> metric_stream.write_point(value, start_time=start, end_time=end) # API call
+
+Stackdriver Monitoring supports several **value types**. A *CUMULATIVE* metric represents a value
+over time. Successive points in a :class:`~gcloud.monitoring.timeseries.TimeSeries` should have the
+same start time and increasing end times, until an event resets the cumulative value to zero and
+sets a new start time for the follow points.
+
+For *GAUGE* metrics, the time intervals a single point in time, so only the end_time should be
+specified::
+
+    >>> metric_stream.write_point(value, end_time=end) # API call
+
+By default, end_time defaults to :meth:`~datetime.datetime.utcnow()`, so metrics written for the
+can be simplified to::
+
+   >>> metric_stream.write_point(value) # API call
+
+For *DELTA* metrics, the points reflect a change during a finite time interval, so both start_time
+and end_time should be specified, though end_time again defaults to the current time.
+
+    >>> metric_stream.write_point(value, start_time=start_time) # API call
+
+As an alternative to :class:`~gcloud.monitoring.timeseries.MetricStream`,  it's also possible
+ to have the :class:`gcloud.monitoring.client.Client`  return a
+ :class:`gcloud.monitoring.timeseries.TimeSeries` object, which can then written later::
+
+    >>> ts1 = client.timeseries(value, end_time=end_time, metric=metric1, resource=resource1)
```

Comment:
I kept them on this pass, if you really feel strongly let me know, but the reasons to keep them are

* Consistency across the library
* Having factory methods lets us keep the argument defaults in there while keeping the namedtuple classes cleaner.